### PR TITLE
fix(date): fix datetime format

### PR DESCRIPTION
**Pull request by thanh-nguyen-dang**
Jira Ticket: [COV-934](https://occ-data.atlassian.net/browse/COV-934)

### Bug Fixes
- IDPH-Vaccine ETL: correct date time format

### CODE REVIEW

File: covid19-etl/etl/idph_vaccine.py [hunk 1]
```diff
@@ -18,6 +18,13 @@
 TOTAL_VACCINE_LINK = "https://idph.illinois.gov/DPHPublicInformation/api/covidvaccine/getStatewideVaccineDetails"
 
 
+def remove_time_from_date_time(str_datetime):
+    datetime_parts = str_datetime.split("T")
+    if len(datetime_parts) > 0:
```
Comment (paulineribeyre) [inline covid19-etl/etl/idph_vaccine.py:23]: the result of `split` will always be of length > 0, so i think you can simplify the function to just `return str_datetime.split("T")[0]`

File: covid19-etl/etl/idph_vaccine.py [hunk 2]
```diff
@@ -215,16 +222,26 @@ def parse_county_data(self):
                 "summary_locations": [{"submitter_id": summary_location_submitter_id}],
             }
             for (key, value) in county_vaccine_mapping.items():
-                if key == "PctVaccinatedPopulation":
+                if value == "vaccine_persons_fully_vaccinated_pct":
                     summary_clinical[value] = int(county_covid_data.get(key) * 100)
-                elif key == "AdministeredCountRollAvg":
+                elif value == "vaccine_administered_count_roll_avg":
```
Comment (paulineribeyre) [inline covid19-etl/etl/idph_vaccine.py:227]: why did these 2 props change?

File: covid19-etl/etl/idph_vaccine.py (hunk 1 above)
Comment (thanh-nguyen-dang) [reply covid19-etl/etl/idph_vaccine.py:23]: just to be sure. The format can change anytime

Comment (paulineribeyre) [reply covid19-etl/etl/idph_vaccine.py:23]: what i meant is that regardless of the values of `str1` and `str2`, `str1.split(str2)` will always contain at least 1 item afaik

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**